### PR TITLE
Update zip module test

### PR DESCRIPTION
**Pull request by YosraAk**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Update install module by zip folder test
| Type?         | refacto 
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | /
| How to test?  | TEST_PATH=full/10_module/1_Zip_module npm run specific-test -- --URL=shopUrl

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/12281)

### CODE REVIEW

File: tests/E2E/test/campaigns/common_scenarios/order.js [hunk 1]
```diff
@@ -200,7 +200,7 @@ module.exports = {
   checkExportedFile: async function (client) {
     await client.downloadCart(ShoppingCart.export_carts_button);
     await client.checkFile(global.downloadsFolderPath, global.exportCartFileName);
-    if (existingFile) {
+    if (global.existingFile) {
```
Comment (PierreRambaud) [inline tests/E2E/test/campaigns/common_scenarios/order.js:203]: Can you do it for all occurrences?

**Review by PierreRambaud — DISMISSED** [review]
existingFile => global.existingFile :)

File: tests/E2E/test/campaigns/common_scenarios/order.js (hunk 1 above)
Comment (YosraAk) [reply tests/E2E/test/campaigns/common_scenarios/order.js:203]: Hi @PierreRambaud,

Ok, i will fix it :) .

Thanks!

**Review by mbadrani — APPROVED** [review]
@ansar21mallouli you can rebase your PR

**Comment by ansar21mallouli** [comment]
@mbadrani done  
Thanks

**Pull request merged**